ssh.1

```diff
@@ -579,6 +579,8 @@ flag),
 (supported message integrity codes),
 .Ar kex
 (key exchange algorithms),
+.Ar gss-kex
+(GSSAPI key exchange algorithms),
```

Comment:
This list seems to be otherwise alphabetically sorted.  Could you move `gss-kex` to the right place in the list?

Reply:
My thinking was that it should come after the kex, since it is is where the people are going to look for this. To accommodate both search-ability and alphabetical ordering, we can rename it to `kex-gss`.

Reply:
Sure.

Reply:
Though it's unfortunate that it goes with `GSSAPIKexAlgorithms` which has things the other way round.  Is searching really a big deal here?  The list is quite short.

Reply:
All the configuration options related to the GSSAPI are prefixed with GSSAPI so it is quite unrelated to this list.

In this list, using `kex-gss` is systematic with other things such as `key-cert` or `cipher-auth`.

The current version implements the changes and adds also a missing header files to prevent some warnings that I saw locally.